assets/javascripts/initializers/ai-bot-replies.js

```diff
@@ -159,13 +161,20 @@ function initializeShareButton(api) {
   };
 
   api.addPostMenuButton("share", (post) => {
-    // very hacky and ugly, but there is no `.topic` in attrs
+    // for backwards compat so we don't break if topic is undefined
+    if (post.topic?.archetype !== "private_message") {
+      return;
+    }
+
     if (
       !currentUser.ai_enabled_chat_bots.any(
         (bot) => post.username === bot.username
       )
     ) {
-      return;
+      // special handling for personas (larger than -1200 means real user)
```

Comment:
```suggestion
      // special handling for personas (persona bot users start at ID -1200 and go down)
```

minor nit, but I don't think calling the ones higher than -1200 real users is accurate, because they are still bots too...just not persona bots :P